src/server/pkg/storage/fileset/index/writer.go

```diff
@@ -149,38 +140,11 @@ func (w *Writer) Close() (retErr error) {
 	for i := 0; i < len(w.levels); i++ {
 		l := w.levels[i]
 		if err := l.cw.Close(); err != nil {
-			return err
+			return nil, err
 		}
 		if l.cw.AnnotationCount() == 1 && l.cw.ChunkCount() == 1 {
 			break
 		}
 	}
-	// Write the final index level to the path.
-	objW, err := w.objC.Writer(w.ctx, w.path)
-	if err != nil {
-		return err
-	}
-	defer func() {
-		if err := objW.Close(); retErr == nil {
-			retErr = err
-		}
-	}()
-	// Handles the empty file set case.
-	if w.root == nil {
-		_, err = pbutil.NewWriter(objW).Write(&Index{})
-		return err
-	}
-	chunk := w.root.DataOp.DataRefs[0].ChunkInfo.Chunk
-	if w.rootTTL > 0 {
```

Comment:
I think we may need to add this back for the time being.  It will be changed with the chunk/gc changes.